string channel_name
int32 cam_id
bool is_active